extensions/machine-learning/src/views/widgets/dashboardWidget.ts

```diff
@@ -237,12 +238,10 @@ export class DashboardWidget {
 					dark: this.asAbsolutePath('images/dark/showLess_inverse.svg'),
 					light: this.asAbsolutePath('images/light/showLess.svg'),
 				};
-				parentPanel.addItem(morePanel, {
-					CSSStyles: morePanelStyle
-				});
+				morePanel.updateCssStyles(Object.assign({}, morePanelStyle, { 'visibility': 'visible' }));
 			} else {
 				showMoreComponent.label = constants.showMoreTitle;
-				parentPanel.removeItem(morePanel);
+				morePanel.updateCssStyles(Object.assign({}, morePanelStyle, { 'visibility': 'hidden' }));
```

Comment:
I do see an accessibility issue, you also have to set the aria-hidden to true/false otherwise screen reader is still able to reach these elements.